cmd/autofix.go

```diff
@@ -0,0 +1,159 @@
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+)
+
+func getAuditFunctions() []interface{} {
+	return []interface{}{
+		auditAllowPrivilegeEscalation, auditReadOnlyRootFS, auditRunAsNonRoot,
+		auditAutomountServiceAccountToken, auditPrivileged, auditCapabilities,
+	}
+}
+
+func runAllAudits(resource k8sRuntime.Object) (results []Result) {
+	for _, function := range getAuditFunctions() {
+		for _, result := range getResults([]k8sRuntime.Object{resource}, function) {
+			results = append(results, result)
+		}
+	}
+	return results
+}
+
+func fixSecurityContextNIL(resource k8sRuntime.Object) k8sRuntime.Object {
```

Comment:
I feel like this function should make an empty context and run each fix function on it, rather than using a set SecurityContext